crates/partition-store/src/protobuf_types.rs

```diff
@@ -1878,13 +1878,15 @@ pub mod v1 {
         impl From<restate_types::invocation::SubmitNotificationSink> for SubmitNotificationSink {
             fn from(value: restate_types::invocation::SubmitNotificationSink) -> Self {
                 let notification_sink = match value {
-                    restate_types::invocation::SubmitNotificationSink::Ingress { request_id } => {
-                        submit_notification_sink::NotificationSink::Ingress(
-                            submit_notification_sink::Ingress {
-                                request_id: Bytes::copy_from_slice(&request_id.to_bytes()),
-                            },
-                        )
-                    }
+                    restate_types::invocation::SubmitNotificationSink::Ingress {
+                        request_id,
+                        ..
+                    } => submit_notification_sink::NotificationSink::Ingress(
+                        submit_notification_sink::Ingress {
+                            unused: Some(GenerationalNodeId::new(1, 1).into()),
```

Comment:
Would be good to leave a comment explaining why `GenerationalNodeId::new(1,1)` is ok here. Why isn't it used in the 1.1?